product-service/package.json

```diff
@@ -13,6 +13,8 @@
     "@middy/core": "^1.4.0",
     "@middy/http-cors": "^1.4.0",
     "ajv": "^6.12.6",
+    "aws-sdk": "^2.797.0",
```

Comment:
AWS lambda environment already contains `aws-sdk`, thus it could be moved to `devDependency` section to reduce bundle size (`aws-sdk` adds ~9mb)
`aws-sdk-mock` also could be moved to `devDependency` section since it needed only for tests